spec/spec_helper.rb

```diff
@@ -1,7 +1,9 @@
-require 'simplecov'
-SimpleCov.start 'rails' do
-  add_filter '/config/'
-  add_filter '/lib/rspec/formatters/user_flow_formatter.rb'
+if ENV['CI']
```

Comment:
what if we called the env var COVERAGE so if we wanted to run it locally, it would be clearer than telling the test suite it's running in CI?